package.json

```diff
@@ -53,6 +53,7 @@
     "intern-geezer": "2.2.3",
     "jscs-jsdoc": "1.1.0",
     "load-grunt-tasks": "3.2.0",
+    "otplib": "7.1.0",
```

Comment:
To sanity-check my understanding, we need this in order to run the tests against a live server, is that accurate?

Reply:
That is correct, it is able to generate a valid code from the secret that is return from the server.